packages/frontend/src/components/ProjectConnection/WarehouseForms/Inputs/StartOfWeekSelect.tsx

```diff
@@ -1,7 +1,8 @@
+import { Select } from '@mantine/core';
 import React, { FC } from 'react';
-import Select2 from '../../../ReactHookForm/Select2';
+import { Controller } from 'react-hook-form';
 
-const defaultOption = { value: null, label: 'Auto' };
+const defaultOption = { value: ' ', label: 'Auto' };
```

Comment:
An empty string will cause issues later on once we enforce API validation.

Let's use `clearable` property instead 👍 
```
                <Select
                    clearable
                    placeholder="Auto"
                    label="Start of week"
                    description="Will be taken into account when using 'WEEK' time interval"
                    data={daysOfWeekOptions}
                    value={field.value}
                    onChange={field.onChange}
                    disabled={disabled}
                    dropdownPosition="top"
                />
```

Reply:
We can remove `const defaultOption = { value: ' ', label: 'Auto' };` now.